portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+var currSlideIndex = 0;
+var slides = document.querySelectorAll(".slide");
+showSlide(currSlideIndex);
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+// Modulus operator (for positive and negative numbers)
+function mod(a,b) {
+    return ((a % b) + b) % b; 
+} 
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+//      
+function advanceSlide(advanceAmount){
+    showSlide(currSlideIndex += advanceAmount);
 }
+
+//
+function showSlide(slideIndex){
+    slides.forEach(slide => slide.style.display = "none");
```

Comment:
In JS we usually use ' and not "
In addition, I think you can define a constant for 'none'. let see what Eyal thinks

Reply:
done :)